generators/app/index.js

```diff
@@ -1,125 +1,122 @@
 'use strict';
+
+/**
+ * Generates the core angular application.
+ */
+
 var Generator = require('yeoman-generator'),
     chalk     = require('chalk'),
     yosay     = require('yosay'),
-    mkdirp    = require('mkdirp'),
-    copydir   = require('copy-dir');
+    caseIt    = require('change-case'),
+    mkdirp    = require('mkdirp');
 
 module.exports = class extends Generator {
-  prompting () {
+  // Initializing Queue
+  initializing () {
     // Have Yeoman greet the user.
-    this.log(yosay(
-      'Welcome to the ' + chalk.red('TricomB2B') + ' generator!'
-    ));
+    this.log(yosay(`Welcome to the ${chalk.red('TricomB2B')} Angular generator!`));
+  }
 
-    var prompts = [{
+  // Prompting Queue
+  prompting () {
+    const prompts = [{
       type: 'input',
       name: 'name',
       message: 'What is the name of your app?',
       default: this.appname
     }, {
       type: 'input',
-      name: 'description',
-      message: 'Description',
-      default: this.appname
-    }, {
-      type: 'input',
-      name: 'primaryColor',
-      message: 'Primary Color',
-      default: '#353535'
-    }, {
```

Comment:
Dropped the primary color input. I think it doesn't add a lot of value, and the spot where it's used would likely be filled in by a favicon generator anyway.